selfdrive/navd/tests/test_map_renderer.py

```diff
@@ -218,12 +211,12 @@ def assert_stat(stat, nominal, tol=0.3):
       tol = (nominal / (1+tol)), (nominal * (1+tol))
       self.assertTrue(tol[0] < stat < tol[1], f"{stat} not in tolerance {tol}")
 
-    assert_stat(_mean,   0.0035)
-    assert_stat(_median, 0.0034)
-    assert_stat(_stddev, 0.00093, tol=0.5)
+    assert_stat(_mean,   0.030)
+    assert_stat(_median, 0.027)
+    assert_stat(_stddev, 0.0078)
 
-    self.assertLess(_max, 0.2)
-    self.assertGreater(_min, 0.0010)
+    self.assertLess(_max, 0.065)
```

Comment:
@mitchellgoffpc is this increased latency a potential issue? what's the range we can tolerate?